open_spiel/games/laser_tag_test.cc

```diff
@@ -132,6 +132,14 @@ void BasicLaserTagTestsBigGrid() {
       10);
 }
 
+void BasicPartiallyObservableLaserTagTests() {
```

Comment:
Please add a test of the observation tensor and observation string. Making the game partially observable isn't really changing the internal game logic, so these tests aren't testing much. If there are bugs, they're likely to show up in the tensor & string.

You don't need to go overboard with the tests, but I would verify that we're getting what we expect in a couple situations, and I'd vary the default params `obs_forward`, `obs_back`, and `obs_side` in the same situations and confirm that works as well.

Reply:
Hey, thanks for the feedback. I realized while reviewing your suggestions that I had forgotten to update the ObservationString methods as well. I have rectified this and added some tests of the observation tensor and observation string as suggested.

Let me know if there is anything I missed or needs changing.